.github/workflows/publish_check.yml

```diff
@@ -43,6 +42,7 @@ jobs:
       - name: Install cargo-release
         run: cargo install cargo-release
       - name: Release (Dry Run)
-        run: cargo release --allow-branch '*' --dry-run -v -p pravega-client-auth -p pravega-client-channel -p pravega-client-config -p pravega-client-macros -p pravega-client-retry -p pravega-client-shared -p pravega-connection-pool -p pravega-controller-client -p pravega -p pravega-wire-protocol
+        run: cargo release --allow-branch '*' --dry-run -v -p pravega-client-shared -p pravega-client-macros -p pravega-client-channel -p pravega-client-retry -p pravega-connection-pool -p pravega-client-config -p pravega-wire-protocol -p pravega-controller-client -p pravega-client-auth
```

Comment:
I think this `--allow-branch` might be needed by the previous `HEAD^`, can we check if `--allow-branch` is needed now?

Reply:
Without `--allow-branch` it complains the following and fails.
`error: cannot release from branch "HEAD", instead switch to "*, !HEAD"`

Reply:
Please the corresponding run here: https://github.com/kotlasaicharanreddy/pravega-client-rust/actions/runs/4955330982/